e2e/glustershd_test.go

```diff
@@ -27,7 +27,8 @@ func TestSelfHeal(t *testing.T) {
 		brickPaths[i-1] = brickPath
 	}
 
-	client := initRestclient(tc.gds[0])
+	client, err := initRestclient(tc.gds[0])
+	r.Nil(err)
```

Comment:
add a new line after `r.Nil(err)`
if we have a new line after each assertion it will be easy to distinguish which assertion belongs to which case

Reply:
can we add a check for `client` also `r.NotNil(client)`